### ISSUE TITLE
Force 

### ISSUE THREAD

**Issue by MikaelHoogen** [basnijholt/adaptive-lighting#59]
So i'm trying this new intregration out and I really like the initiative. All my lights are controlled by the circadian lightning. Good work with this one!

There is one thing I must have in order to make the switch. I need the possibility to force the lights to be set by CT/mired/kelvin like you can in circadian lightning. I have a couple of IKEA color lights, they don't support to set CT/mired/kelvin by native, but I have them in deCONZ and that integration is making the translation so you can set CT/mired/kelvin in HA. I don't get the translation HA does to RGB/hs/xy, it's horrible and don't look even close to what it is supposed to. deCONZ make the translation perfektly, looks exact correct.

**Comment by basnijholt** [comment]
Hi @MikaelHoogen,

I do not understand your problem. In what mode do you need the lights to be? CT, mired, or Kelvin?

Perhaps you need the `prefer_rgb_color` option? https://github.com/basnijholt/adaptive-lighting#options

**Comment by MikaelHoogen** [comment]
Hello @basnijholt 

I want to force it to either CT, mired, or Kelvin. I don't think it makes any difference.

The `prefer_rgb_color` doesn't do anything on a IKEA color bulb. I think it's because HA doesn't repport it as a supported feature. supported_features: 61. But you can send a on command with kelvin and deCONZ translates it correctly to the bulb.

So my wish would be a `force_kelvin` option.

**Comment by RubenKelevra** [comment]
Hey @MikaelHoogen,

HA released some changes on how lights support color temperatures and rgb values. Can you verify that you still have trouble using adaptive lighting?

**Comment by MikaelHoogen** [comment]
> Hey @MikaelHoogen,
> 
> HA released some changes on how lights support color temperatures and rgb values. Can you verify that you still have trouble using adaptive lighting?

Yeah. The changes made it even worse. Since then I can't use circadian lightning either. So my IKEA rgb bulbs are pretty useless :( :(

**Comment by RubenKelevra** [comment]
Hey @MikaelHoogen,

I can hear your frustration - I'm sorry.

I'm moving this/next month, but I hope to find time soon to contribute to this project. Since IKEA-bulbs are one of the stuff on my buy list they will be one of the focuses.  😁

**Comment by th3w1zard1** [comment]
@MikaelHoogen Are you still having this issue in the latest version 1.7.0?

**Comment by MikaelHoogen** [comment]
> @MikaelHoogen Are you still having this issue in the latest version 1.7.0?

Hi, sorry but I don't use any IKEA rgb anymore. Gave them away.
